my-dsl.md

```diff
@@ -1,38 +1,40 @@
 # Language
-_What is the name of the language? Link the name to its webpage 
-(if appropriate)._
+Alda
+[Source Code](https://github.com/alda-lang/alda)
 
 
 # Domain
-_Describe the language's domain in five words._
+General Light-weight Music Programming Language
 
 
 # Computational model
-_What is the underlying computational model of this language? To answer this 
-question, provide a high-level description (no more than 100 words) of the 
-computation that occurs when someone executes a program in this language._
-
+When someone runs an alda script, their code is first converted into Clojure
+code which creates a map of the score. That is then used to produce sounds 
+that the alda code described. 
 
 # DSL-ness
-_Fowler writes about a spectrum of languages, from general-purpose languages to 
-"purely" domain-specific. Where does the DSL you chose fall on this spectrum, 
-and why?_ 
-
+Alda is purely domain-specific. It only allows for the description of different 
+lines of music. There are no loops or conditionals, so the language is not
+turing complete. There is no computation that can be done with the language
+other than playing static lines of music. 
```

Comment:
By static, I assume you mean that a single Alda script will only ever produce a single result. But, are there things like macros to make it easy to put the same part in multiple locations? Is there an easy way to change the key of a whole section? Are there any conveniences like these, or is Alda basically a textual score?